src/tests/UglifyJSPlugin.test.ts

```diff
@@ -0,0 +1,40 @@
+import { BundleProducer } from "../core/BundleProducer";
+import { FuseBox } from "../index";
+import { should } from "fuse-test-runner";
+import { Bundle } from "../core/Bundle";
+import { createEnv } from "./stubs/TestEnvironment";
+import { UglifyJSPlugin } from "../plugins/UglifyJSPlugin";
+const getTestEnv = require("./fixtures/lib").getTestEnv;
+
+const file = `
+var longVar = 'str1';
+var longVar2 = 'str2';
+
+module.exports = function () {return longVar + ' ' + longVar2;}
+`;
+
+export class UglifyJSPluginTest {
+  "Should return compressed js 1"() {
+      return getTestEnv({
```

Comment:
all of the other commits for the last 10 days do not have passing tests, not an easy testing environment to change it with. It was converted to run here, it will be removed, but this is half way.